client/express-checkout/blocks/index.js

```diff
@@ -0,0 +1,22 @@
+/* global wcpayConfig */
+
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT } from '../../checkout/constants';
+import { ExpressCheckout } from './express-checkout';
+import { ApplePayPreview } from './apple-pay-preview';
```

Comment:
By updating how that component is exported, we need to update the import too.

```suggestion
import ApplePayPreview from './apple-pay-preview';
```